fideslib/oauth/oauth_util.py

```diff
@@ -46,7 +46,11 @@ def verify_oauth_client(
         extract_payload(authorization, config.security.app_encryption_key)
     )
 
-    issued_at = token_data.get(JWE_ISSUED_AT, None)
+    try:
+        issued_at = token_data.get(JWE_ISSUED_AT, None)
+    except exceptions.JWEParseError as exc:
+        raise AuthorizationError(detail="Not Authorized for this action") from exc
```

Comment:
The error happens at the load.

```suggestion
    try:
        token_data = json.loads(
            extract_payload(authorization, config.security.app_encryption_key)
        )
    except exceptions.JWEParseError as exc:
        raise AuthorizationError(detail="Not Authorized for this action") from exc

    issued_at = token_data.get(JWE_ISSUED_AT, None)

```

Reply:
yep accidentally wrapped the wrong thing and didn't test before pushing

Reply:
Do you just run your tests in a virtual environment locally for fideslib, @sanders41? I haven't successfully gotten tests running for fideslib yet, they start and then perpetually hang for the file included here.